### PR TITLE
Allow resize for chat

### LINKED ISSUE #3144: High priority - The chatbox could be stretched(as one possible solution)
@minotogna 

**Is your feature request related to a problem? Please describe.**
**Hard to read**:
The messages would be folded 1. if there is a link, the user has to move the scroll bar, which is not convenient and can cover some information if they want to see the full sentence. 

2. or simply if it is a long text writing in the text box(#3019).

**Describe the solution you'd like**
The chatbox could be stretched

**Priority**
High

### PR DESCRIPTION
**Pull request by sorja**
New library: `re-reselect`

https://github.com/openforis/fra-platform/assets/5508251/95a06c2b-89d3-4721-a811-33c6732db56a

https://github.com/openforis/fra-platform/assets/5508251/d293c259-316a-4fb0-84bc-b4d8b0b2fe34

### CODE REVIEW

File: src/client/components/MessageCenter/Topic/Message/Message.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const Message: React.FC<MessageProps> = (props) => {
           </div>
         </div>
       </div>
-      {!deleted && <div className="message-body">{message.message}</div>}
+      {!deleted && <p className="message-body">{message.message}</p>}
```
Comment (minotogna) [inline src/client/components/MessageCenter/Topic/Message/Message.tsx]: why p?

File: src/client/components/Resizable/Resizable.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+type Props = React.PropsWithChildren<ResizableProps>
+
+const Resizable: React.FC<Props> = (props: Props) => {
+  const { children, className, maxWidth, minWidth } = props
+
+  return (
+    <ReResizable maxWidth={maxWidth} minWidth={minWidth} className={className}>
+      {children}
```
Comment (minotogna) [inline src/client/components/Resizable/Resizable.tsx]: always better to use React.Children.toArray(children)

**Comment by sorja** [comment]
Check vertical resize
allow resize top left right and corners

**Review by minotogna — APPROVED** [review]
👍👍

**Pull request merged**